src/item/EndCrystal.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\item;
+
+use pocketmine\block\Block;
+use pocketmine\block\BlockTypeIds;
+use pocketmine\entity\Location;
+use pocketmine\entity\object\EndCrystal as EntityEndCrystal;
+use pocketmine\math\AxisAlignedBB;
+use pocketmine\math\Facing;
+use pocketmine\math\Vector3;
+use pocketmine\player\Player;
+use function count;
+
+class EndCrystal extends Item{
+
+	public function onInteractBlock(Player $player, Block $blockReplace, Block $blockClicked, int $face, Vector3 $clickVector, array &$returnedItems) : ItemUseResult{
+		if($blockClicked->getTypeId() === BlockTypeIds::OBSIDIAN || $blockClicked->getTypeId() === BlockTypeIds::BEDROCK){
+			$pos = $blockClicked->getPosition();
+			$world = $pos->getWorld();
+			$bb = AxisAlignedBB::one()
```

Comment:
This should be a 2x2x2 BB because of recent vanilla changes. You can no longer place an ender crystal next to one.

Reply:
Really ? ... :/

Reply:
If you mean that there need to be a distance in 1 block between 2 crystals then the current code works fine since the crystal spawns at the centered position

Reply:
1. End crystal
2. Air
3. End crystal
That's how it should be (imagine it horizontally). The end crystal does spawn centered but that BB isn't centered.

Reply:
After further testing, it seems I got confused and the BB isn't centered in the block corner, it *starts* from the corner and the 1x1x2 BB that starts inside the block is somehow the correct behavior in vanilla, which I thought it was a bug here, but seems like a bug in bedrock's side